packages/release-mastermind/src/utils/labels.ts

```diff
@@ -6,80 +1,88 @@
+/** @format */
+
+import { LoggingDataClass, LoggingLevels } from "@videndum/utilities"
+import { Utils } from "."
+import { log } from ".."
+import { Labels, Runners } from "../action"
+import { formatColor } from "./parsingData"
 
 /**
  * Syncronise labels to repository
  * @author IvanFon, TGTGamer, jbinda
  * @since 1.0.0
  */
-export async function sync(this: Utils, config: Runners['labels']) {
-  /**
-   * Syncronises the repo labels
-   * !todo Add delete labels
-   * @since 2.0.0
-   */
-  if (!config) throw new Error('Cannot syncronise labels without config')
-  const curLabels: Labels = await this.api.labels.get().catch(err => {
-    throw log(
-      LoggingLevels.error, `Error thrown while getting labels: ` + err)
-  })
-  log(LoggingLevels.debug, `curLabels: ${JSON.stringify(curLabels)}`)
-  for (const configLabel of Object.values(config)) {
-    const label = curLabels[configLabel.name.toLowerCase()]
+export async function sync(this: Utils, config: Runners["labels"]) {
+	/**
+	 * Syncronises the repo labels
+	 * !todo Add delete labels
+	 * @since 2.0.0
+	 */
+	if (!config) throw new Error("Cannot syncronise labels without config")
+	const curLabels: Labels = await this.api.labels.get().catch((err) => {
+		throw log(LoggingLevels.error, `Error thrown while getting labels: ` + err)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
    throw log(LoggingLevels.error, "Error thrown while getting labels: " + err)
```